MULTITENANCY.md

```diff
@@ -0,0 +1,123 @@
+kind: Secret
+metadata:
+  creationTimestamp: "2022-01-24T08:40:02Z"
+  name: lb-csi-creds
+  namespace: kube-system
+  resourceVersion: "1837602"
+  uid: 406c3583-d4cd-4a0b-8615-3a6f2b9b7577
+
+```
+
+## Lightbits and NVMEoTCP
+
+Once a volume is created and mounted, e.g. a PVC and PV, the csi driver will first create the volume on the lightos API with his token and set the hosts which are allowed to talk to this volume to the name of the worker nodes.
+
+```bash
+k get pvc,pv,node
+NAME                                     STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+persistentvolumeclaim/data-sampla-app-0  Bound    pvc-c4b7822b-b3c8-414a-a1fa-9350d30a4f5c   1Gi        RWO            partition-silver   25s
+
+NAME                                                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                        STORAGECLASS       REASON   AGE
+persistentvolume/pvc-c4b7822b-b3c8-414a-a1fa-9350d30a4f5c   1Gi        RWO            Delete           Bound    sampla-app/data-sampla-app-0 partition-silver            25s
+
+NAME                                               STATUS   ROLES   AGE     VERSION
+node/shoot--pd76mr--inttest0-group-0-845b8-49r7x   Ready    node    6d21h   v1.21.9
+node/shoot--pd76mr--inttest0-group-0-845b8-ng7xh   Ready    node    7d1h    v1.21.9
+```
+
+The pod wich mounts this volume is running on the node `shoot--pd76mr--inttest0-group-0-845b8-49r7x`.
+
+```bash
+k get pod -o wide
+NAME          READY   STATUS    RESTARTS   AGE     IP              NODE                                          NOMINATED NODE   READINESS GATES
+sampla-app-0  1/1     Running   0          5m34s   10.244.15.152   shoot--pd76mr--inttest0-group-0-845b8-49r7x   <none>           <none>
+```
+
+on the Lightos cluster has set the ACL of this volume to be mounted only from the node where the pod is running and trying to mount this volume.
+
+```bash
+lbcli list volumes 
+Name                                       UUID                                   State       Protection State   NSID      Size      Replicas   Compression   ACL                                                                                               Rebuild Progress
+pvc-c4b7822b-b3c8-414a-a1fa-9350d30a4f5c   7828aa17-2316-442d-883e-d000436d41f2   Available   FullyProtected     631       1.0 GiB   2          true          values:"nqn.2019-09.com.lightbitslabs:host:shoot--pd76mr--inttest0-group-0-845b8-49r7x.node"      None
+```
+
+The NVMEoTCP implementation in the linux kernel on the worker node side and on the lightos side implement the name of the volume to match these ACL expectations.
```

Comment:
"the implementation ... implement", missing an "s" but don't know if formulation is weak anyway (pleonasm?)